src/main/java/mate/controller/AddDriverToCarController.java

```diff
@@ -0,0 +1,39 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Car;
+import mate.model.Driver;
+import mate.service.CarService;
+import mate.service.DriverService;
+
+public class AddDriverToCarController extends HttpServlet {
+    private static final String PACKAGE = "mate";
+    private static final String PATH_TO_JSP = "/WEB-INF/views/cars/drivers/addDriversToCar.jsp";
```

Comment:
```suggestion
    private static final String PATH_TO_JSP = "/WEB-INF/views/cars/drivers/add.jsp";
```
in order to make your jsp names consistent with urls